metadata-ingestion/src/datahub/ingestion/source/tableau/tableau_server_wrapper.py

```diff
@@ -0,0 +1,27 @@
+from tableauserverclient import Server, UserItem
+
+from datahub.ingestion.source.tableau import tableau_constant as c
+
+
+class LoggedInUser:
+    user: UserItem
+    _site_id: str
+
+    def __init__(self, server: Server):
```

Comment:
a user should call `LoggedInUser.from_server(server)` - the init method should be kept simple